.github/workflows/build.yml

```diff
@@ -38,10 +38,11 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           context: .
+          platforms: linux/amd64
           push: true
           tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.VERSION }}, ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest
-          cache-from: type=registry,ref=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest
-          cache-to: type=registry,ref=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest,mode=max
+          cache-from: type=registry,ref=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:buildcache
+          cache-to: type=registry,ref=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:buildcache,mode=max
```

Comment:
The `cache-from` and `cache-to` which we now use with label/tag `buildcache` as I understand it just makes it faster to do a build instead of having to pull the base image every stinking time. In other words, I think GitHub's Container Registry will have our previous image layers locally, and whenever the first few layers are identical (majority case), it will be much faster to re-use them.